src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel;
+
+public final class UserRepository {
+
+    public User getContactUsUser() {
```

Comment:
when you will start using lombok, here will be possible to use builder, make it static

Reply:
done